components/Content/CaseStudies.jsx

```diff
@@ -38,15 +40,13 @@ export const CaseStudies = ({ limit }) => {
         </h2>
 
         <div className="grid gap-8 md:grid-cols-2 lg:grid-cols-3">
-          {(limit ? caseStudies.slice(0, limit) : caseStudies).map(
-            (blogItem) => (
-              <Article
-                key={blogItem.id}
-                article={blogItem}
-                href={`/blog/${blogItem?.attributes?.slug}`}
-              />
-            ),
-          )}
+          {(limit ? items.slice(0, limit) : items).map((blogItem) => (
+            <Article
+              key={blogItem.id}
```

Comment:
@atepem seems you misunderstood me, all was good with `const caseStudies = data?.data ?? [];` I meant these should be 'caseStudy' or 'item', 

<img width="599" alt="image" src="https://github.com/user-attachments/assets/9d9f91e8-ab19-4e37-a8e5-2c2c821f7875">

Reply:
ohh my bad! should be a quick fix 😅